script-base.js

```diff
@@ -291,7 +338,7 @@ Generator.prototype.addGradleDependency = function (scope, group, name, version)
             file: fullPath,
             needle: 'jhipster-needle-gradle-dependency',
             splicable: [
-                scope + ' group: \'' + group + '\', name: \'' + name + '\', version: \'' + version + '\''
+                    scope + ' group: \'' + group + ', name: \'' + name + ', version: \'' + version + '\''
```

Comment:
can you review this ? see https://github.com/jhipster/generator-jhipster/pull/2479